bbin/git.py

```diff
@@ -10,10 +10,10 @@
 GIT = shutil.which("git") or str(Path("/usr/bin/git"))
```

Comment:
Change this to just `GIT = shutil.which("git")` or else the if statement would not be able to be reached

Reply:
I thought I already did. Changing it now.